'docker-compose up' fails with Build error

When executing docker-compose-up, it fails with the error "There is no feature named VERSION_ORDERING_V2". 

kayenta_1  | Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details
kayenta_1  | 
kayenta_1  | FAILURE: Build failed with an exception.
kayenta_1  | 
kayenta_1  | * Where:
kayenta_1  | Settings file '/tmp/workdir/settings.gradle' line: 17
kayenta_1  | 
kayenta_1  | * What went wrong:
kayenta_1  | A problem occurred evaluating settings 'workdir'.
kayenta_1  | > There is no feature named VERSION_ORDERING_V2
kayenta_1  | 
kayenta_1  | * Try:
kayenta_1  | Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
kayenta_1  | 
kayenta_1  | * Get more help at https://help.gradle.org
kayenta_1  | 
kayenta_1  | BUILD FAILED in 3s
kayenta_kayenta_1 exited with code 1

This is because the dockerfile uses an out of date version of gradle, bumping the image to the following will get around this issue:
`FROM gradle:6.5-jdk11 AS builder`
(the newer java runtime is needed to compile one of the components)

but after doing this, I still run into an issue where the integration tests don't pass:

```
7 tests completed, 7 failed                                                                                            
                                                                                                                       
> Task :kayenta-integration-tests:test FAILED
                                                           
FAILURE: Build failed with an exception.           
```

I have been able to successfully build and start kayenta using the `build-debug` branch, though, so might be worth using that instead for the time being

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,9 @@ COPY . /tmp/workdir
 WORKDIR /tmp/workdir
 
 # Build kayenta
-RUN gradle build
+# Doesn't run build because the integration tests try to spin up a Docker
+# container to talk to a Kayenta instance, which requires some Docker shenanigans.
+RUN gradle assemble
 
 # Unpack so release image can copy folder and be smaller
 RUN tar -xf /tmp/workdir/kayenta-web/build/distributions/kayenta.tar -C /opt

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
 #
 # Builder Image
 #
-FROM gradle:5.4-jdk8 AS builder
+FROM gradle:6.5-jdk11 AS builder
 
 # Prep build environment
 ENV GRADLE_USER_HOME=cache
@@ -21,7 +21,7 @@ FROM alpine:3.11
 
 LABEL maintainer="delivery-engineering@netflix.com"
 
-RUN apk --no-cache add --update openjdk8-jre
+RUN apk --no-cache add --update openjdk11-jre
 
 # Set where to look for config from
 ENV KAYENTA_OPTS=-Dspring.config.location=file:/opt/kayenta/config/kayenta.yml

